Reduce dev friction

iman-nebulr opened the pull request.

iman-nebulr committed:
Custom params 2 (build fix) (#71) (#72)

* Fixes for dockerfile

* App resolver and service updated to match new nblocks-ts-client

custom param configs and custom params on tokens

added function in app.service call
Automatic formatting and linting by post commit hook

Fixed some compilation errors in files I have not altered.

added script to build with watch

* More flexible local package installation destination with host.docker.internal

* Fixed nullable property

* updating node version

* fixed casing in dockerfile

* package lock held internal package references

---------

Co-authored-by: orr3 <o.soderlund@gmail.com>
Co-authored-by: Oscar Söderlund <oscar@nebulr.group>

iman-nebulr committed:
updating dependencies

iman-nebulr committed:
Package updates and adaptions to go along with these

iman-nebulr committed:
nebulr-config-module up to date

iman-nebulr committed:
setting versions

iman-nebulr committed:
Merge branch 'master' into reduce-dev-friction

iman-nebulr committed:
removing overrides

orr3 commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nebulr-group/nblocks-nestjs",
-  "version": "5.2.1",
+  "version": "5.3.0-beta.1",
```
Since you're leaving support for older versions in many of the core dependencies I'd argue the next version should be 6.0.0-beta.1

iman-nebulr replied:
1..here is an additional pull request for stage.
2. i will change to 6.0

orr3 reviewed (commented):
Change target branch to `stage` because this is what you want right?

iman-nebulr committed:
removing husky, updating package-lock

iman-nebulr committed:
missing dependencies during build

iman-nebulr committed:
Reduce dev friction (#74)

* Custom params 2 (build fix) (#71)

* Fixes for dockerfile

* App resolver and service updated to match new nblocks-ts-client

custom param configs and custom params on tokens

added function in app.service call
Automatic formatting and linting by post commit hook

Fixed some compilation errors in files I have not altered.

added script to build with watch

* More flexible local package installation destination with host.docker.internal

* Fixed nullable property

* updating node version

* fixed casing in dockerfile

* package lock held internal package references

---------

Co-authored-by: Iman Pouya <iman@nebulr.group>
Co-authored-by: Oscar Söderlund <oscar@nebulr.group>

* updating dependencies

* Package updates and adaptions to go along with these

* nebulr-config-module up to date

* setting versions

* removing overrides

* removing husky, updating package-lock

* missing dependencies during build

---------

Co-authored-by: orr3 <o.soderlund@gmail.com>
Co-authored-by: Oscar Söderlund <oscar@nebulr.group>

iman-nebulr committed:
synced resourcemappings with parent projectswq

iman-nebulr committed:
package lock updates

iman-nebulr committed:
Merge remote-tracking branch 'origin/stage' into reduce-dev-friction
